### ISSUE TITLE
New Photos View wrapping preview icons

### ISSUE THREAD

**Issue by HankG** [diaspora/diaspora#7942]
I thought we tested this case but maybe we didn't.  User over on Diasp.org running 0.7.8.0 is having this artifact in their photo viewer.  https://diasp.org/posts/10957047
![image](https://user-images.githubusercontent.com/1683691/49453808-da631a00-f7b1-11e8-8ade-06722245e8d2.png)

**Comment by HankG** [comment]
Here's the link to the page so can be more easily reproduced.  Just click on a photo.  https://diasp.org/people/055ca690bf5601362a51047d7b62795e/photos

**Comment by Flaburgan** [comment]
That's kinda beautiful but probably not what the user is expecting ^^ 
This is because I designed that pictures viewer for posts which rarely contain more than a dozen of pictures, but I forgot we also you use it on the photos tab. I'll think about it and propose a patch.

**Comment by goobertron** [comment]
Perhaps we can just revert to the previous UI for the photos stream? On 0.7.6.0, with several hundred images, it looks like this:

![photostream](https://user-images.githubusercontent.com/2530168/49454687-9423b000-f7dd-11e8-88b6-cff90da25a03.png)

**Comment by Flaburgan** [comment]
IMO thumbnails with this size are useless, we could choose to simply hide them. But a scroll-x is also possible.

**Comment by HankG** [comment]
@goobertron I think the fix to keep it consistent won't be too bad so we should be able to role it out as easily as doing the revert.  @Flaburgan can comment on that more intelligently than I can though.

**Comment by Flaburgan** [comment]
Here is how an horizontal scroll could look like. What do you think?
![capture du 2018-12-04 23-13-52](https://user-images.githubusercontent.com/930064/49476480-6446cd80-f81a-11e8-96c8-f16233fd9334.png)

**Comment by goobertron** [comment]
That looks lovely. Is scrolling done via the little arrow at the bottom right of the screen? If so, it could be larger/more prominent and horizontally centred on the thumbnails. If it's nothing to do with scrolling, ignore that comment. There's a scroll-bar under the thumbnails and I assume you can also 'swipe' the thumbnails themselves, which would be enough.

**Comment by HankG** [comment]
> Here is how an horizontal scroll could look like. What do you think?
> ![capture du 2018-12-04 23-13-52](https://user-images.githubusercontent.com/930064/49476480-6446cd80-f81a-11e8-96c8-f16233fd9334.png)

Yep looks good.

**Comment by HankG** [comment]
FYI to people testing, the scrolling of the thumbnails is not the arrow (which is a slideshow player) but the scrollbar.  This looks good to me in my testing.

**Comment by Flaburgan** [comment]
>Is scrolling done via the little arrow at the bottom right of the screen?

Yes as Hank said the arrow is for playing the photos. It is there for a while but I never noticed it. That should probably be improved for sure, even if I'm not sure many would use it.
